recipes/natural_language_processing/summarizer/bootc/Containerfile

```diff
@@ -4,7 +4,8 @@
 # from recipes/natural_language_processing/summarizer, run
 # 'make bootc'
 
-FROM quay.io/centos-bootc/centos-bootc:stream9
+#FROM quay.io/centos-bootc/centos-bootc:stream9
+FROM registry.redhat.io/rhel9-beta/rhel-bootc:9.4
```

Comment:
I don't think this should be the default for an upstream project, because this container image is not available to everyone.